### DISCUSSION TITLE
Custom user control

### DISCUSSION THREAD

**Discussion by jfre11** [vanjarosoftware/Vanjaro.Platform#976] [General]
I have several custom user controls modules.   They seem to be working as expected on a Vanjaro page -- except for one of them.    I'm having a issue when I drag and drop  this custom app  onto a blank page into a grid.  It includes the dnn persona bar -(see attachment)  inside the grid.   The ascx control is simple -- has no javascript.  But when I publish the page  everything gets  pushed into about 2 inches to the right. (see attachment)

  I'm sure its something in my module, but I stripped all java script and css from the module--that's why its in discussion.  Again -- it works fine when I drop it on a standard DNN page.    I've attached screen shots with the code.  Also, no errors in the error log either 
[VJ_issue.docx](https://github.com/vanjarosoftware/Vanjaro.Platform/files/6179251/VJ_issue.docx)

**Comment by mksmanish79** [comment]
Can you share the module with us so that we can duplicate the issue or provide us url for the site where its deployed so that we can have a look. You can send the details directly to manish.kumar@mandeeps.com.

**Comment by jfre11** [comment]
The Vanjaro page freezes when I add a custom control to the page.     I can't run in  Edit mode.    The custom control works fine on a normal DNN page.   I'm trying to test via  localhost  on my PC.  I sent Manish my code.

Here's the error log:
2021-04-15 23:26:43,905 [JFDell][Thread:73][FATAL] DotNetNuke.Web.Common.Internal.DotNetNukeHttpApplication - System.Web.HttpException (0x80004005)
   at System.Web.Util.FileUtil.CheckSuspiciousPhysicalPath(String physicalPath)
   at System.Web.CachedPathData.ValidatePath(String physicalPath)
   at System.Web.HttpContext.ValidatePath()
   at System.Web.HttpApplication.PipelineStepManager.ValidateHelper(HttpContext context)

 Any idea ?
